.changelog/11998.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+Add examples for using go-sockaddr with multiple interfaces
+```
```

Comment:
```suggestion
```
Changelogs are not needed for docs/website copy changes. You can ignore the check failure by adding the `pr/no-changelog` label, and then delete this file.